spec/requests/healthcheck_spec.rb

```diff
@@ -0,0 +1,8 @@
+RSpec.describe "/healthcheck" do
+  it "returns database connection status" do
+    get "/healthcheck"
+    json = JSON.parse(response.body)
+
+    expect(json["checks"]).to include("database_connectivity")
```

Comment:
Probably worth testing the status is OK and/or the HTTP response code is 200, like the other apps have been doing. Otherwise this is looking great 👍

Reply:
Done